schema/dcschema_enum_instances.mcf

```diff
@@ -123,11 +123,1601 @@ description: "Category of race or ethnicity group used by Opportunity Insights t
 typeOf: dcs:OI_RaceCodeEnum
 
 ###---- begin BEA enum instances ----###
-Node: dcid:ConsumerGoodsAndServices
-name: "ConsumerGoodsAndServices"
+Node: dcid:BLS_SEFV04
 typeOf: dcs:EconomicProductEnum
-description: "Consumer goods are usually tangible final commodities purchased by the average consumer. Consumer services are intangible activities that are typically produced by other people and consumed by the consumer simultaneously."
-descriptionUrl: "https://en.wikipedia.org/wiki/Goods_and_services"
+description: "Food from vending machines and mobile vendors"
+
+Node: dcid:BLS_SAF11
+typeOf: dcs:EconomicProductEnum
+description: "Food at home"
```

Comment:
Please add a name to these enums, similar to http://browser.datacommons.org/browser/NAICS/92

These are helpful in StatVar and StatPop displays: https://datacommons.org/browser/Count_Worker_NAICSAccommodationFoodServices

Reply:
Ok, I will turn the descriptions to names

Reply:
done. I moved them to the bottom of the file